apache-hbase-mixin/dashboards/apache-hbase-regionserver-overview.libsonnet

```diff
@@ -0,0 +1,1309 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'apache-hbase-regionserver-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local regionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'server_region_count{job=~"$job", hbase_cluster=~"$hbase_cluster", instance=~"$instance"}',
+      datasource=promDatasource,
+      legendFormat='{{instance}}',
+      format='time_series',
+    ),
+  ],
+  type: 'stat',
```

Comment:
The RegionServer overview I am a little concerned with scalability but I am not familiar with how many instances are expected in an environment. I notice you have 3 stat panels that may scale weirdly if you have multiple instances and clusters. How many instances are expected with hbase at a time and would it make sense to change that to a scrollable table showing the instance and values for the first 2 panels, and maybe horizontal scrollable bar for the `Store file size` chart?

Reply:
I can definitely make the bar gauge horizontal, but the rest of the panels in the top row were specifically requested as stat panels by Grafana.

Reply:
I just did a quick check, and saw that you might start with a minimum of three RegionServer instances for fault tolerance and scalability. A larger clusters may have tens or even hundreds of RegionServer instances. So that dashboard may only be usable when selecting for a small amount of instance anyways.